### DISCUSSION TITLE
MGPUSIM multi-GPU simulation

### DISCUSSION THREAD

**Discussion by wwwuxy** [sarchlab/mgpusim#101] [Q&A] [answered]
Does the MGPUSIM platform simulate multiple GPUs, or does it simulate multiple cores within a single GPU? If it is the former, is it possible to simulate the scenario of multiple cores within a single GPU by modifying the inter-GPU communication latency?

**Comment by syifan** [comment, marked as answer]
MGPUSim does both. It simulates multiple cores within each GPU of multi-GPU platforms.

**Reply by Yzgll** [reply]
Can I schedule cores in each GPU？

**Reply by syifan** [reply]
Can you be a bit more specific on core scheduling?
